mnefun/_mnefun.py

```diff
@@ -610,20 +610,26 @@ def fetch_raw_files(p, subjects, run_indices):
         raw_dir = op.join(subj_dir, p.raw_dir)
         if not op.isdir(raw_dir):
             os.mkdir(raw_dir)
-        finder_stem = 'find %s ' % p.acq_dir
-        # build remote raw file finder
-        fnames = get_raw_fnames(p, subj, 'raw', True, False, run_indices[si])
+        fnames = get_raw_fnames(p, subj, 'raw', True, False,
+                                run_indices[si])
         assert len(fnames) > 0
-        finder = (finder_stem +
+        # build remote raw file finder
+        assert type(p.acq_dir) in (list, string_types)
+        for remote_dir in p.acq_dir:
+            finder_stem = 'find %s ' % remote_dir
+            finder = (finder_stem +
                   ' -o '.join(['-type f -regex ' + _regex_convert(f)
                                for f in fnames]))
-        stdout_ = run_subprocess(['ssh', '-p', str(p.acq_port),
-                                  p.acq_ssh, finder])[0]
+            stdout_ = run_subprocess(['ssh', '-p', str(p.acq_port),
+                                      p.acq_ssh, finder])[0]
+            if len(stdout_) > 0:
+                break
```

Comment:
This `break` method will only work if the files are only in one location, not if they are spread across multiple locations. This can be fixed with something like:
```
remote_fnames = list()
for remote_dir in use_dirs:
    ...
    these_fnames = [x.strip() for x in stdout_.splitlines()]
    assert all(fname.startswith(remote_dir + '/') for fname in these_fnames)
    remote_fnames += these_fnames
```
But really this is even a bit clunky, since you can [make find search multiple directories at the same time] (http://alvinalexander.com/blog/post/linux-unix/find-how-search-multiple-folders-directories-unix). That should be the cleanest solution, and will probably require the fewest changed lines (at least here).